src/libopensc/types.h

```diff
@@ -374,6 +374,8 @@ typedef struct sc_serial_number {
 	struct sc_iin iin;
 } sc_serial_number_t;
 
+typedef int sc_change_counter;
```

Comment:
I think this typedef is a little too much. Just using a size_t to reflect the state would be enough, I think.